test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -440,6 +442,363 @@ spec setup =
             (x * Nothing).to_vector . should_equal nulls
             (x / Nothing).to_vector . should_equal nulls
 
+    Test.group prefix+"Rounding-like operations" <|
+        Test.specify "should name a rounding column correctly" <|
+            table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+            table.at "x" . round . name . should_equal "round([x])"
+
+        test_floatlike type =
+            Test.specify "should allow round on a "+type.to_text+" column" <|
+                table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+                result = table.at "x" . cast type . round
+                result.to_vector.should_equal [0, 1, 3, 4, 0, -1, -3, -4]
+                result.value_type . is_integer . should_be_true
+
+            Test.specify "should allow round on a float column (to >0 decimal places)" <|
+                table = table_builder [["x", [0.51, 0.59, 3.51, 3.59, -0.51, -0.59, -3.51, -3.59]]]
+                result = table.at "x" . cast type . round 1
+                result.to_vector.should_equal [0.5, 0.6, 3.5, 3.6, -0.5, -0.6, -3.5, -3.6]
+                result.value_type . is_integer . should_be_false
+
+            Test.specify "should allow truncate on a "+type.to_text+" column" <|
+                table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+                result = table.at "x" . cast type . truncate
+                result.to_vector.should_equal [0, 0, 3, 3, 0, 0, -3, -3]
+                result.value_type . is_integer . should_be_true
+
+            Test.specify "should allow ceil on a "+type.to_text+" column" <|
+                table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+                result = table.at "x" . cast type . ceil
+                result.to_vector.should_equal [1, 1, 4, 4, 0, 0, -3, -3]
+                result.value_type . is_integer . should_be_true
+
+            Test.specify "should allow floor on a "+type.to_text+" column" <|
+                table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+                result = table.at "x" . cast type . floor
+                result.to_vector.should_equal [0, 0, 3, 3, -1, -1, -4, -4]
+                result.value_type . is_integer . should_be_true
+
+        test_floatlike Value_Type.Float
+        test_floatlike Value_Type.Decimal
```

Comment:
I don't think this will work in-memory - we don't implement a dedicated Decimal type - IMO this should predicated with a check if the backend supports decimal type (SQLite does not support it either - so we should run this part of tests only for Postgres).

Reply:
Other than that - a nice way to implement 2 suites using a common logic! Cool :D

Reply:
Ah, I didn't realize this test suite ran against the in-memory table too. Why did it pass, if two backends don't support Decimal?

Reply:
Because how `cast` is constructed.

If a requested target type is not supported but a similar type is supported, the backend falls back to the similar supported type and just issues a warning `Inexact_Type_Coercion` telling about the coercion.

For example see the following for an SQLite table:
<img width="908" alt="image" src="https://github.com/enso-org/enso/assets/1436948/9e6ae0dc-cae4-400c-90ce-8adcd9235051">

Interestingly, for in-memory tables it indeed fails with:
```
>>> (Error: (Illegal_Argument.Error 'Columns of type Decimal (precision=Nothing, scale=Nothing) are currently not supported in the in-memory backend.' Nothing))
```

So I'm not sure how were the tests passing. Were you running the full test suite?

Reply:
I wasn't, mostly I've been running PG and SL tests, so that's why I didn't see a failure.

Reply:
Done.